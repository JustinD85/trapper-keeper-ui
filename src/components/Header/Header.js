import React from "react";
import { Link } from "react-router-dom";
import { withStyles } from '@material-ui/core/styles'
import { AppBar, Typography, IconButton, Tooltip } from '@material-ui/core'
import AddIcon from '@material-ui/icons/Add'

const styles = {
  toolTip: {
    maxWidth: 100
    
  },

}

const Header = ({ classes }) => (
  <AppBar>
      <Typography variant='h1' align='center'>Trapper-Keeper</Typography>
      <Tooltip title='Add a new note!' placement='bottom-end' className={classes.toolTip}><Link to="/new-note"><IconButton color='secondary'><AddIcon/></IconButton></Link></Tooltip>
  </AppBar>
);

export default withStyles(styles)(Header)