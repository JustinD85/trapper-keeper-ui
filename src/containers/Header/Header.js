import React from "react";
import { Link } from "react-router-dom";

export const Header = () => (
  <div className="Header">
    <h1 className="title">Trapper-Keeper</h1>
    <Link to="/new-note">Add Note</Link>
  </div>
);

export default Header;
