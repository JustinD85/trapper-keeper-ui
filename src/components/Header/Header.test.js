import React from "react";
import Header from "./Header";
import { shallow } from "enzyme";

it("should render without crashing", () => {
  let wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
