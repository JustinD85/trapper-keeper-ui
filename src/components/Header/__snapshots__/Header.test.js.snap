// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without crashing 1`] = `
<div
  className="Header"
>
  <h1
    className="title"
  >
    Trapper-Keeper
  </h1>
  <Link
    replace={false}
    to="/new-note"
  >
    Add Note
  </Link>
</div>
`;
